src/workspace/qml/MuseScore/Workspace/internal/WorkspacesView.qml

```diff
@@ -34,6 +34,10 @@ RadioButtonGroup {
     boundsBehavior: Flickable.StopAtBounds
     orientation: Qt.Vertical
 
+    ScrollBar.vertical: ScrollBar {
```

Comment:
I think it's better to use our `StyledScrollBar` component instead, which will have the correct look and behaviour.

Reply:
Hi @cbjeukendrup, thanks for pointing this out. 
I have updated the QML file accordingly and amended the commit with the new changes. I noticed that it appears with a different commit ID. If you want me remove this pull request and start from scratch that also fine for me.

Reply:
Thank you, it is perfect! We prefer amending existing PRs instead of closing and creating new ones, to keep the PR history clean. The different commit ID is no problem (the content of the commit is indeed different, so it will have a different ID). 

And congratulations on having your first PR merged!

Reply:
Thanks @cbjeukendrup. I hope I can contribute with other things in the upcoming weeks.
Keep the great work  !